### DISCUSSION TITLE
Optionally display the save button at the bottom of forms in addition to the top.

### DISCUSSION THREAD

**Discussion by calebuharrison** [avo-hq/avo#671] [Feature requests]
When a resource has a lot of fields and the edit form view is very long, it can be easy to forget to scroll back up to the top to hit save. It would be nice if resources had a configuration option that allowed for optionally displaying the save button at the bottom of the form as well.

As a bonus, it might be nice to alert users when they're navigating away from a form that has unsaved changes.

**Comment by adrianthedev** [comment]
+1 for both suggestions. Will schedule them.

https://github.com/avo-hq/avo/issues/672
https://github.com/avo-hq/avo/issues/673

**Comment by adrianthedev** [comment]
Hey @nulldotpro. Released this in [2.1.0](https://github.com/avo-hq/avo/releases/tag/v2.1.0)
